### PR TITLE
Update and fix testing

### PR DESCRIPTION
**Pull request by urso**
Change fixes recent test failures for ARM32 architecture

Updates:
- enable go modules in CI
- update go cross compiler version to go1.13
- fix ARM cross compilation compiler versions to 1.12/1.13
- Update mage and magetools dependencies in go.mod (fix go modules)
- List dependencies on controller machine, not target machine
  - fix go modules downloading and trying to test dependencies (used to
    break testing on arm)
- fix format string 'Run test for package' not being printed correctly
- remove appveyor tests in favor of more up-to date travis based windows tests

### CODE REVIEW

**Comment by ph** [comment]
Still a bit RED in the CI?

File: magefile.go [hunk 1]
```diff
@@ -179,7 +181,7 @@ func Test() error {
 			tst := goRun.Test
 			bin := path.Join(home, path.Base(pkg))
 			useBinary := fs.ExistsFile(bin) && testUseBin
-			fmt.Println("Run test for package '%v' (binary: %v)", pkg, useBinary)
+			fmt.Printf("Run test for package '%v' (binary: %v)\n", pkg, useBinary)
```
Comment (kvch) [inline magefile.go]: Nit: why not use `%s` for substituting `pkg` which is a string?

Comment (urso) [reply magefile.go]: What is the advantage of using `%s` over `%v`? When passing primitive types the fmt package uses the same code path for `%v` and `%s`.

Comment (kvch) [reply magefile.go]: No advantage. I guess it's just a personal preference. :)

**Pull request merged**